src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -704,104 +710,114 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                     break;
                 }
                 case 3:
+                    assert(!HWIntrinsicInfo::IsFmaIntrinsic(intrinEmbMask.id) || (falseReg != embMaskOp3Reg));
```

Comment:
`assert(falseReg != embMaskOp3Reg)` seemed overzealous for these new APIs. I was hitting it before moving it off the path of those APIs, and this doesn't seem to cause any issues?